packages/o-spreadsheet-engine/src/helpers/text_helper.ts

```diff
@@ -58,7 +61,38 @@ export function getDefaultContextFont(
   return `${italicStr} ${weight} ${fontSize}px ${DEFAULT_FONT}`;
 }
 
-const textWidthCache: Record<string, Record<string, number>> = {};
+export function computeMultilineTextWidth(
```

Comment:
this function could be removed and replaced by `computeMultilineTextSize(context, text, style, fontUnit).width` ?